sass/main.scss

```diff
@@ -754,10 +815,306 @@ footer {
     }
 }
 
-
-@media screen and (max-width: 1630px) {
+@media screen and (max-width: 1200px) {
 
     #leadership #leaders {
+        grid-template-columns: repeat(4, $leadership-size-mobile);
         justify-content: space-between;
     }
 }
+
+@media screen and (max-width: 900px) {
+
+    // Redefine only for mobile
+    $nav-height: 60px;
+
+    h1, section > p {
+        text-align: center;
+    }
+
+    h1 {
+        font-size: 2.5em !important; // FIXME
+    }
+
+    p {
+        font-size: 1em;
+    }
+
+    .section-pad {
+        padding: 40px;
+    }
+
+    .section-pad-top {
+        padding-top: 30px;
+    }
+
+    .section-pad-left {
+        padding: 20px;
+    }
+
+    nav {
+        height: $nav-height;
+
+        #logo img {
+            width: $nav-height;
+            height: $nav-height;
+        }
+
+        #mlh-badge {
+            width: 90px;
+        }
+
+        .link, #login {
+            display: none;
```

Comment:
Users are not allowed to login if they are on their phones?

I remember chris mentioning confusion with the name of the other button being register

Consider fitting it in the nav or adding one below/beside register to avoid users confused about where to access their login portal